### PR TITLE
 added SCTV in BuonMaThuot, DakLak, Vietnam

### PR DESCRIPTION
**Pull request by dnmnhat**
added SCTV in BuonMaThuot, DakLak, Vietnam

### CODE REVIEW

**Comment by Flole998** [comment]
Please squash your commits together

**Comment by dnmnhat** [comment]
@Flole998 I'm new to github, so there are issues about using github I'm not very clear, but for the above request I suggest 2 files about the frequency of SCTV TV in my country, SCTV in my country. My country uses both DVB systems, DVBC and DVBT2. There should be 2 frequency files, one for DVBC and one for DVBT2.
I don't understand what you mean???
Can you tell me ???
Thanks

**Comment by Flole998** [comment]
You have 3 commits with different changes, including a merge commit. There should be a single commit with all your changes.

**Comment by dnmnhat** [comment]
let me try again 
Thanks
